modules/@apostrophecms/ui/ui/apos/components/AposButton.vue

```diff
@@ -110,6 +117,17 @@ export default {
     tabindex() {
       return this.disableFocus ? '-1' : '0';
     },
+    colorStyle() {
+      if (this.type === 'color') {
+        let style = ''
+        style += `background-color: ${this.color};`;
```

Comment:
A lot better — how about:

```
return {
  backgroundColor: this.color,
  border: `2px solid ${tinycolor(this.color).lighten(20).toString()};`
};
```

Which saves you a bunch of of the string building? (Vue will automatically do the right thing if you bind an object with js-style css property names to :style)